frodo.js

```diff
@@ -1,63 +1,66 @@
-/**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!